src/examples/mfem_common_plane.cpp

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#include <set>
+
+#ifdef TRIBOL_USE_UMPIRE
+// Umpire includes
+#include "umpire/ResourceManager.hpp"
+#endif
+
+// MFEM includes
+#include "mfem.hpp"
+
+// Axom includes
+#include "axom/CLI11.hpp"
+#include "axom/slic.hpp"
+
+// Redecomp includes
+#include "redecomp/redecomp.hpp"
+
+// Tribol includes
+#include "tribol/common/Parameters.hpp"
+#include "tribol/config.hpp"
+#include "tribol/interface/tribol.hpp"
+
+/// Simple central difference method.
+class CentralDiffSolver : public mfem::SecondOrderODESolver
+{
+public:
+  CentralDiffSolver(const mfem::Array<int>& bc_vdofs_)
```

Comment:
Should these be in Tribol namespace?

Reply:
They're in the `mfem_ext` namespace now.